internal/pkg/install/install.go

```diff
@@ -190,6 +191,13 @@ func RunInstallerContainer(disk, platform, ref string, cfg configcore.Config, cf
 		}
 	}
 
+	if !slices.Contains(options.ExtraKernelArgs, "console=ttyS0") {
+		// preserve console=ttyS0 if it was already present in cmdline
+		if procfs.ProcCmdline().Get("console").Contains("ttyS0") {
+			args = append(args, "--extra-kernel-arg", "console=ttyS0")
```

Comment:
I think we should better move this into the installer itself, this code runs on the host